tests/envs/test_action_dim_check.py

```diff
@@ -48,12 +48,16 @@ def test_mujoco_action_dimensions(env_spec: EnvSpec):
     env.close()
 
 
-@pytest.mark.parametrize(
-    "env",
+DISCRETE_ENVS = list(
     filter(
         lambda env: isinstance(env.action_space, spaces.Discrete),
```

Comment:
@pseudo-rnd-thoughts So, uh, this has been merged before I got a chance to look at it, but a more pythonic/readable/better way would probably be to use a list comprehension, so instead of 
`list(filter(lambda env: isinstance(env.action_space, spaces.Discrete), all_testing_initialised_envs))`
it would be
`[env for env in all_testing_initialised_envs if isinstance(env.action_space, spaces.Discrete)]`

Reply:
That is a good point, sorry I sent it to Jordan as well because I thought the code was mostly good. I will see if I can fix this